src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,134 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final int ARRAYS_ZERO_BORDER = 0;
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public MyLinkedList() {
+        head = new Node<>(null, null, null);
+        tail = new Node<>(null, null, null);
```

Comment:
i am not sure that you need this constructor :)

Reply:
Thank you, i checked this) May be i could solve it in another way, but my code don't work with out it))

Reply:
you don't need this. you can pass `null` as parameters in nodes you add